src/ast/mod.rs

```diff
@@ -1114,6 +1114,7 @@ pub enum Statement {
         columns: Vec<Ident>,
         query: Box<Query>,
         with_options: Vec<SqlOption>,
+        cluster_by: Vec<Ident>,
```

Comment:
https://docs.snowflake.com/en/sql-reference/sql/create-materialized-view.html says

```
  [ CLUSTER BY ( <expr1> [, <expr2> ... ] ) ]
```

But this PR has cluster by as only columns (not expressions).

I am fine with merging this as is but I wonder if you want to support the more general syntax?